src/dmd/dsymbolsem.d

```diff
@@ -3311,24 +3311,31 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                     funcdecl.introducing = 1;
                     if (cd.classKind == ClassKind.cpp && Target.reverseCppOverloads)
                     {
-                        // with dmc, overloaded functions are grouped and in reverse order
+                        /* Overloaded functions with same name are grouped and in reverse order.
```

Comment:
What about DMC? the previous comment mentions it. Does it no longer apply to just DMC?

Reply:
I figured the `Target.reverseCppOverload` on line 3312 covered that.

Reply:
OK.